src/Fortify/Console/Commands/RunPlaybookCommand.php

```diff
@@ -104,14 +104,17 @@ private function askPlaybookName(array $availablePlaybooks): string
 
     private function getAvailablePlaybooks(): array
     {
+        /**
+         * @var array
+         */
         $files = scandir(__DIR__.'/../Playbooks');
 
         unset($files[0], $files[1]);
 
         return array_map(fn (string $file) => str_replace('.php', '', $file), $files);
     }
 
-    private function resolvePlaybookDefinition($class): PlaybookDefinition
+    private function resolvePlaybookDefinition(mixed $class): PlaybookDefinition
```

Comment:
@alexbarnsley I prefer using `mixed` as a last resort. What do you think if we use union type instead?

```suggestion
    /** @param PlaybookDefinition|Playbook|class-string $class */
    private function resolvePlaybookDefinition(PlaybookDefinition|Playbook|string $class): PlaybookDefinition
```